Sources/Penny/CommandsManager.swift

```diff
@@ -91,46 +91,36 @@ enum SlashCommand: String, CaseIterable {
         switch self {
         case .howManyCoinsApp:
             return .user
-        case .link, .autoPings, .faqs, .howManyCoins:
+        case .github, .autoPings, .faqs, .howManyCoins:
             return nil
         }
     }
 }
 
-enum LinkSubCommand: String, CaseIterable {
-    case discord, github, slack
+enum GitHubSubCommand: String, CaseIterable {
+    case help
+    case link
+    case unlink
 
     var description: String {
         switch self {
-        case .discord:
-            return "Link your Discord account"
-        case .github:
-            return "Link your GitHub account"
-        case .slack:
-            return "Link your Slack account"
+        case .help:
+            return "Help about how GitHub linking works"
+        case .link:
+            return "Link your GitHub account to Penny"
+        case .unlink:
+            return "Unlink your GitHub account from Penny"
         }
     }
 
     var options: [ApplicationCommand.Option] {
         switch self {
-        case .discord: return [.init(
-            type: .string,
-            name: "id",
-            description: "Your Discord account ID",
-            required: true
-        )]
-        case .github: return [.init(
-            type: .string,
-            name: "id",
-            description: "Your GitHub account ID",
-            required: true
-        )]
-        case .slack: return [.init(
-            type: .string,
-            name: "id",
-            description: "Your Slack account ID",
-            required: true
-        )]
+        case .help:
+            return []
+        case .link:
+            return []
+        case .unlink:
+            return []
```

Comment:
you can just clamp them all into the same line:
```suggestion
        case .help, .link, .unlink:
            return []
```